DNN Platform/DotNetNuke.DependencyInjection/Extensions/TypeExtensions.cs

```diff
@@ -1,23 +1,25 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
 namespace DotNetNuke.DependencyInjection.Extensions
 {
     using System;
     using System.Linq;
-    using System.Reflection;
+    using System.Reflection;
 
+    using DotNetNuke.Instrumentation;
+
     /// <summary>
     /// <see cref="Type"/> specific extensions to be used
     /// in Dependency Injection invocations.
     /// </summary>
     public static class TypeExtensions
-    {
-        // There is no logging in this file by design as
-        // it would create a dependency on the Logging library
-        // and this library can't have any dependencies on other
-        // DNN Libraries.
-
+    {
+        // There is no logging in this file by design as
+        // it would create a dependency on the Logging library
+        // and this library can't have any dependencies on other
+        // DNN Libraries.
+

```

Comment:
@bdukes 

Do we:
🅰️ Remove this comment and accept having that dependency on DotNetNuke.Instrumentations
🅱️ There is a reason for not having that dependency and we need to think about another solution ?

Reply:
I vote A.Ah missed that comment. There is no reason not to have a dependency there. The DI library was not referenced any where else and we already had a logger available. I can remove the comment.

Reply:
I agree, it's just not my expertise and I don't know (did not dig into it yet) who put the comment there, so would like to get input from @bdukes and/or @ahoefling  maybe on this...

Reply:
Also since there was already an ILog object in the DotNetNuke.DependencyInjection project that library already had a dependency on DotNetNuke.Instrumentation.

Reply:
Oh, then should we adjust the .nuspec file to declare this dependency here https://github.com/dnnsoftware/Dnn.Platform/blob/develop/Build/Tools/NuGet/DotNetNuke.DependencyInjection.nuspec @donker @mitchelsellers ?

Reply:
Where is that other ILog @mtrutledge ?

Reply:
Line 19 of DependencyInjectionInitialize.cs

Reply:
Awesome so my last concern is just about if we need to adjust the nuspec file @donker  ?